modules/coreos-layering-configuring-on-revert.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * machine_configuration/coreos-layering.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="coreos-layering-configuring-on-revert_{context}"]
+= Reverting an on-cluster custom layered image
+
+You can revert an on-cluster custom layered image from nodes by removing the label for the machine config pool (MCP) that you specified in the `MachineOSConfig` object. After you remove the label, the Machine Config Operator (MCO) reboots the nodes in that MCP with the cluster base {op-system-first} image, along with any previously-made machine config changes, overriding the custom layered image. 
+
+[IMPORTANT]
+====
+If the node where the custom layered image is deployed uses a custom machine config pool, before you remove the label, make sure the node is associated with a second MCP.
+====
+
+You can reapply the custom layered image to the node by using the `oc label node/<node_name> 'node-role.kubernetes.io/<mcp_name>='` label.
+
+.Prerequisites
+
+* You have opted-in to on-cluster layering by creating a `MachineOSConfig` object.
+
+.Procedure
+
+* Remove the label from the node by using the following command:
++
+[source,terminal]
+----
+$ oc label node/<node_name> node-role.kubernetes.io/<mcp_name>-
+----
++
+When you save the changes, the MCO drains, cordons, and reboots the nodes. After the reboot, the node uses the cluster base {op-system-first} image.
+
+.Verification
+
+You can verify that the custom layered image is removed by performing any of the following checks:
```

Comment:
This sounds like you only need to do one, is that the case? If so, I'd make the below items unnumbered bullets.

Or if they aren't, then I'd remove "any" here

```suggestion
You can verify that the custom layered image is removed by performing the following checks:
```